### PR TITLE
Improve ProjectNode::addDetails

### PR DESCRIPTION
**Pull request by majetideepak**
The ProjectNode::addDetails method has been improved to not include a trailing comma.

### CODE REVIEW

File: velox/core/PlanNode.cpp [hunk 1]
```diff
@@ -108,11 +108,13 @@ namespace {
 void addKeys(
     std::stringstream& stream,
     const std::vector<std::shared_ptr<const FieldAccessTypedExpr>>& keys) {
+  bool isNotFirstKey = false;
   for (auto i = 0; i < keys.size(); ++i) {
-    if (i > 0) {
+    if (isNotFirstKey) {
```
Comment (mbasmanova) [inline velox/core/PlanNode.cpp:113]: perhaps, use 'i' and drop isNotFirstKey variable?

`if (i > 0)`

File: velox/core/PlanNode.cpp [hunk 2]
```diff
@@ -126,11 +128,13 @@ void AggregationNode::addDetails(std::stringstream& stream) const {
     stream << "] ";
   }
 
+  bool isNotFirstName = false;
   for (auto i = 0; i < aggregateNames_.size(); ++i) {
-    if (i > 0) {
+    if (isNotFirstName) {
```
Comment (mbasmanova) [inline velox/core/PlanNode.cpp:133]: What's the motivation for this change?

Comment (majetideepak) [reply velox/core/PlanNode.cpp:133]: I feel this has better code readability. But if you find it to be verbose, I can revert it. 
I have seen a similar pattern here. https://github.com/facebookincubator/velox/blob/main/velox/vector/SelectivityVector.h#L300

Comment (mbasmanova) [reply velox/core/PlanNode.cpp:133]: I feel (i > 0) is easier to read. In the example you shared "i" doesn't work.

Comment (majetideepak) [reply velox/core/PlanNode.cpp]: Done.

**Review by mbasmanova — APPROVED** [review]
@majetideepak Thank you, Deepak. This reads better. I also appreciate you moving code into .cpp file as it helps speed up incremental builds.
